servicemanager/servicemanager.go

```diff
@@ -264,34 +368,142 @@ func (sm *ServiceManager) doApplyService(service ServiceInfo) (err error) {
 			continue
 		}
 
-		if err := os.RemoveAll(oldService.ImagePath); err != nil {
+		if _, err := sm.removeService(oldService); err != nil {
 			log.Errorf("Can't remove old service: %s", err)
 		}
+	}
+
+	service.Timestamp = time.Now().UTC()
```

Comment:
No need to update timestamp, it will be updated by UseService API

Reply:
Done